server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -982,6 +983,8 @@ public synchronized void close() throws IOException {
         toClose.add(injector.getInstance(RemoteClusters.class));
         toClose.add(() -> stopWatch.stop().start("logical_replication_service"));
         toClose.add(injector.getInstance(LogicalReplicationService.class));
+        toClose.add(() -> stopWatch.stop().start("shard_replication_service"));
+        toClose.add(injector.getInstance(ShardReplicationService.class));
```

Comment:
a lot of failures with 
org.elasticsearch.common.inject.ProvisionException: Guice provision errors:

```
1) Error injecting constructor, java.lang.IllegalArgumentException: Setting is not registered for key [replication.logical.ops_batch_size]
  at io.crate.replication.logical.LogicalReplicationSettings.<init>(Unknown Source)
  while locating io.crate.replication.logical.LogicalReplicationSettings
    for parameter 2 at io.crate.replication.logical.ShardReplicationService
```
will move `logicalReplicationSettings()` to the `EsTestCase`